src/argowrapper/constants.py

```diff
@@ -47,14 +47,14 @@
 
 
 class POD_COMPLETION_STRATEGY(Enum):
-    ONWORKFLOWSUCCESS: Final = "OnWorkflowSuccess"
-    ONPODSUCESS: Final = "OnPodSuccess"
-    ONPODCOMPLETION: Final = "OnPodCompletion"
+    ONWORKFLOWSUCCESS = "OnWorkflowSuccess"
```

Comment:
Why are we removing Final here?

Reply:
Because type annotations doesn't work in Enums: https://github.com/microsoft/pyright/issues/8543

Reply:
@pieterlukasse thoughts? I think we need the final to prevent the code trying to change these constant variables?